tools/Customization/DevHome.Customization/Views/OptimizeDevDriveDialog.xaml

```diff
@@ -18,10 +18,14 @@
     SecondaryButtonText="Cancel">
     <StackPanel>
         <TextBlock Text="{x:Bind ViewModel.OptimizeDevDriveDialogDescription}" TextWrapping="Wrap"/>
-        <Grid Margin="0 10 0 0">
-            <TextBox HorizontalAlignment="Left" MinWidth="300" Grid.Column="0" x:Name="DirectoryPathTextBox" PlaceholderText="Choose a directory" Text="{x:Bind ViewModel.DirectoryPathTextBox , Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"/>
+        <Grid Margin="0 10 0 0" >
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition />

```

Comment:
Nit; prefer to be explicit with the width here. I don't know if the default is "auto" or "*".

Reply:
Default is "*"